laravel/app/Console/Commands/InitiateEval.php

```diff
@@ -105,152 +106,114 @@ public function initResidentEvalAttendingPOST($evaluatorID, $evaluateeID)
+                    }
                 }
+            } else {
+                Log::debug("Cannot initiate evaluation because no resident ID for " . $pair["residentName"] . " or no attending ID for " . $pair["attendingName"] . " can be found.");
             }
-            // either the resident of the attending werent valid, mark it down -- used for testing only
-            if ($evalSent == 0) {
-                //print($evalID[$i].' '.$evaluateeID.' '.$evaluatorID);
+        }
+        Log::debug("Total Number of Evaluations Succesfully sent: " . $totalEvalsSent);
+    }
+
+    public function getResidentAttendingPairs($date)
+    {
+        $pairs = array();
+        //get residents that have evaluation data in DB on day
+        $residents = EvaluateData::where("date", $date)->pluck("resident_id", "resident");
+        Log::info("Residents in Evaluation Data with date " . $date . ":  " . $residents);
+        // for each resident get their active rotation service and create pairs of residents and attendings that worked together
+        foreach ($residents as $residentName => $residentID) {
+            //get Resident's active rotation service
+            $activeService = 0;
+            $activeRotation = Rotations::where("name", $residentName)->where("Start", "<=", $date)->where("End", ">=", $date)->first();
+            if (is_null($activeRotation)) {
+                Log::info("Could not find an active rotation for Resident " . $residentName . " ID: " . $residentID);
+            } else {
+                $activeService = $activeRotation["Service"];
+            }
+            Log::info("Resident " . $residentName . " has active service id " . $activeService);
+            $attendings = EvaluateData::where("date", $date)->where("resident_id", $residentID)->pluck("attending_id", "attending")->unique();
+            foreach ($attendings as $attendingName => $attendingID) {
+                $overallTime = EvaluateData::where("date", $date)->where("resident_id", $residentID)->where("attending_id", $attendingID)->sum("time_with_attending");
+                Log::info("Resident: " . $residentName . "(" . $residentID . ") Attending: " . $attendingName . "(" . $attendingID . ") Total Time Together: " . $overallTime);
+                $residentAndAttending = array(
+                    "residentName" => $residentName,
+                    "residentID" => $residentID,
+                    "residentActiveService" => $activeService,
+                    "attendingID" => $attendingID,
+                    "attendingName" => $attendingName,
+                    "overallTime" => $overallTime,
+                );
+                array_push($pairs, $residentAndAttending);
             }
         }
-        Log::debug($evalSent . " evaluations sent");
+        return $pairs;
     }
 
     public function handle()
```

Comment:
Since the `handle` method is the main entry point to `InitiateEval.php`, I'd expect it to be at the top. Since it calls `initiateEvaluations`, I'd expect that to be next, etc.